include/ccmath/math/compare/isgreater.hpp

```diff
@@ -24,7 +27,11 @@ namespace ccm
 	template <typename T>
 	constexpr bool isgreater(T x, T y) noexcept
```

Comment:
Shouldn't you check for integer and float types there? I think (for sanity sake) add a check for types.

Reply:
Ok fixed the functions and also made them better follow the actual standard.